theano/gof/cc.py

```diff
@@ -910,6 +910,12 @@ def support_code(self):
         The support code from Variables is added before the support code from Ops.This might contain duplicates.
         """
         ret = []
+        if config.cmodule.debug:
```

Comment:
I don't like having only one global state for this. But I don't see a clean/easy work around.